src/test/java/integration/Config.java

```diff
@@ -88,4 +88,19 @@ void imageDefault() throws IOException {
         imageDefaultOutput,
         CoreMatchers.containsString(versionOutput.trim()));
   }
+
+  @Test
+  @DisplayName("configured format is respected")
+  void format() throws IOException {
+    String configListOutputFileName = "Config_format_stdout";
+    List<String> commands = new ArrayList<>();
+    commands.add("terra config set format json");
+    commands.add("terra config list > " + configListOutputFileName);
+    int exitCode = TestBashScript.runCommands(commands, Collections.emptyMap());
+    assertEquals(0, exitCode, "config list executed successfully");
+    String configListOutput =
+        Files.readString(
+            TestBashScript.getOutputFilePath(configListOutputFileName), StandardCharsets.UTF_8);
+    assertThat("config list is in JSON format", configListOutput, CoreMatchers.containsString("{"));
```

Comment:
I'm concerned about this assertion returning false positives in the future. Can you change it to either check that the whole thing's a valid JSON object or maybe check a larger string that includes the property name : value?

Reply:
done